dask/dataframe/tests/test_groupby.py

```diff
@@ -554,3 +554,32 @@ def test_groupby_multiprocessing():
     with dask.set_options(get=get):
         assert_eq(ddf.groupby('B').apply(lambda x: x),
                   df.groupby('B').apply(lambda x: x))
+
+
+@pytest.mark.parametrize('agg_func', ['sum', 'var', 'mean', 'count', 'size',
+                                      'std', 'nunique', 'min', 'max'])
+def test_aggregations_multilevel(agg_func):
+    def call(g, m, **kwargs):
+        return getattr(g, m)(**kwargs)
+
+    pdf = pd.DataFrame({'a': [1, 2, 6, 4, 4, 6, 4, 3, 7] * 10,
+                        'b': [4, 2, 7, 3, 3, 1, 1, 1, 2] * 10,
+                        'c': [0, 1, 2, 3, 4, 5, 6, 7, 8] * 10},
+                       columns=['c', 'b', 'a'])
+
+    ddf = dd.from_pandas(pdf, npartitions=10)
+
+    assert_eq(call(pdf.groupby(['a', 'b'])['c'], agg_func),
+              call(ddf.groupby(['a', 'b'])['c'], agg_func, split_every=2))
+
+    assert_eq(call(pdf.groupby(['a'])['c'], agg_func),
+              call(ddf.groupby(['a'])['c'], agg_func))
+
+    assert_eq(call(pdf.groupby(pdf['a'])['c'], agg_func),
+              call(ddf.groupby(ddf['a'])['c'], agg_func))
+
+    assert_eq(call(pdf.groupby([pdf['a'], pdf['b']])['c'], agg_func),
+              call(ddf.groupby([ddf['a'], ddf['b']])['c'], agg_func))
+
+    assert_eq(call(pdf.groupby([pdf['a'] > 2, pdf['b'] > 1])['c'], agg_func),
+              call(ddf.groupby([ddf['a'] > 2, ddf['b'] > 1])['c'], agg_func))
```

Comment:
It might be nice to extend one of the existing tests to include multi-series groupers.  For example you could add new groupers to the for loop in `test_split_apply_combine_on_series`.